src/theme/CodeBlock/CopyButton/index.js

```diff
@@ -0,0 +1,61 @@
+import React, { useCallback, useState, useRef, useEffect } from 'react';
+import clsx from 'clsx';
+// @ts-expect-error: TODO, we need to make theme-classic have type: module
+import copy from 'copy-text-to-clipboard';
+import { translate } from '@docusaurus/Translate';
+import styles from './styles.module.css';
+export default function CopyButton({ code, className }) {
+  const [isCopied, setIsCopied] = useState(false);
+  const copyTimeout = useRef(undefined);
+  const handleCopyCode = useCallback(() => {
+    const removeLineNumbers = /^\s*\d+\s/gm;
```

Comment:
Do the line numbers always have one or more spaces preceding the digits?

Reply:
@jasongitmail this is always the case with the current formatting of the code snippets with line numbers on all the tutorials. When there are single and double digit line numbers on the same code snippet, Double digit numbers have 1 preceding space and single digit numbers have 2 preceding spaces. I assume the spacing will be similar when there are double digit and triple digit numbers on the same code snippet. I can create another quick pr to handle all edge cases when there possibly are no preceding spaces.